extension/chrome/elements/attachment_preview.ts

```diff
@@ -79,9 +80,11 @@ View.run(
           });
           $('#attachment-preview-download')
             .css('display', 'flex')
-            .on('click', e => {
+            .on('click', async e => {
               e.stopPropagation();
-              Browser.saveToDownloads(attachmentForSave);
+              if (!this.attachment.isExecutableFile() || (await Ui.modal.confirm(Lang.attachment.executableFileWarning))) {
```

Comment:
Wouldn't it be simpler to call the helper method `AttachmentWarnings.confirmSaveToDownloadsIfNeeded`  without the `confirmationResultTracker` parameter? It will have the same effect

Reply:
The way we did it in another place:
```
        if (await AttachmentWarnings.confirmSaveToDownloadsIfNeeded(attachment)) {
          Browser.saveToDownloads(attachment);
        }
```